cnv/cnv_users_guide/cnv-monitoring-vm-health.adoc

```diff
@@ -0,0 +1,17 @@
+[id="cnv-monitoring-vm-health"]
+= Monitoring virtual machine health
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-monitoring-vm-health
+toc::[]
+
+Liveness probes stop the `VirtualMachineInstance` if it fails and allow
+controllers such as `VirtualMachine` or `VirtualMachineInstanceReplicaSet` to spawn
+other `VirtualMachineInstances`, in order to restore responsiveness to the
+virtual machine.
+
+Readiness probes are indicators for Services and Endpoints that
+the `VirtualMachineInstance` is ready to receive traffic from Services.
+If readiness probes fail, the `VirtualMachineInstance` will be removed from the
+Endpoints, which back services until the probe recovers.
```

Comment:
This is missing a comma? "... which back Services**,** "

I might be inclined to re-phrase this sentence.

If readiness probes fail, the `VirtualMachineInstance` will be removed from applicable Endpoints until the probe recovers.

Reply:
Thanks @stu-gott --- yes, that's much better. Something struck me as not quite right about this either, but without the clarity of the intention it escaped me. The way I had it worded,  Endpoints was the subject but as the which clause is not needed, that clears things up a lot!

Done.